lib/Test/Nginx/Socket.pm

```diff
@@ -2993,6 +3003,23 @@ C<--- raw_request>.
 
 See also the L<TEST_NGINX_USE_HTTP2> system environment for the "http2" test mode.
 
+=head2 curl_protocol
+
+Set protocol (such as http/https) when sending request using 'curl'.
+
+=head2 curl_options
+
+Add extra command line options when using 'curl' to send request.
+
+Below example will send https request via curl:
+
+--- http2
```

Comment:
The sample code block should start with the tab

Reply:
update